src/steps/smart-campaign-add-lead.ts

```diff
@@ -35,12 +58,21 @@ export class AddLeadToSmartCampaignStep extends BaseStep implements StepInterfac
         return this.error("Can't add %s to %s: found %d matching campaigns", [email, campaign, campaigns.length]);
       }
 
-      const lead: any = await this.client.findLeadByEmail(email);
-      const result = await this.client.addLeadToSmartCampaign(campaigns[0].id.toString(), lead.result[0]);
-      if (result.success) {
-        return this.pass('Successfully added lead %s to smart campaign %s', [email, campaign]);
+      const campaignRecord = this.keyValue('campaign', 'Smart Campaign', campaigns[0]);
+
+      const leadResponse: any = await this.client.findLeadByEmail(email);
```

Comment:
It looks like the response of this call when no fields are provided vs. when fields are provided differs, which leads to different data being displayed.  The data that comes with this step is 💯, while the data displayed on the lead check step is ...  Very sparse.

![Screen Shot 2020-02-11 at 1 59 40 PM](https://user-images.githubusercontent.com/3496491/74283806-6a1a2780-4cd7-11ea-91b9-b38aa103a3c5.png)

- [ ] Can we update the `findLeadByEmail` method to automatically include `createdAt`, `updatedAt`, `firstName`, and `lastName`, in addition to any specific requested field?
- [ ] We should also update all instances of the `lead` ExpectedRecord definition (across all relevant steps) to include those guaranteed fields.

And one minor idea: perhaps we vary the record name based on success/failure?  E.g. if the step passed, name the record `Lead Added` and if the step did not pass, name it `Lead To Be Added`

Reply:
^ I implemented the non-checklisted idea as well